server/apps/main/templates/main/home.html

```diff
@@ -20,135 +18,156 @@ <h1 class="big-heading">AutSPACEs</h1>
+      </a>
     </div>
-      <a class="col-lg-12" href="{{ auth_url }}">
-       <button class="btn btn-outline-light btn-lg float-right platform-button" type="button">Read More</button>
-   </a>
+
+    <div class="col platform-box usage-box">
+      {% if user.is_authenticated %}
+      <a href="{% url 'main:my_stories' %}">
+      {% else %}
+      <a href="{% url 'main:registration' %}">
+      {% endif %}
+        <h3 class="usage-heading">
+          Edit
+          <br/>
+          <i class="usage-icon m-5 bi bi-pencil-square"></i>
+        </h3>
+      </a>
     </div>
 
+    <div class="col platform-box usage-box">
+      <a href="{% url 'main:what_autism_is' %}">
+        <h3 class="usage-heading">
+          Learn
+          <br/>
+          <i class="usage-icon m-5 bi bi-mortarboard"></i>
+        </h3>
+      </a>
+    </div>
+  </div>
 </section>
-<!--  View Experiences -->
+
+<!-- View Experiences -->
 <section id="experience">
   <div class="row">
-      <div class="col-lg-6 shared-experience">
-      <h2 class="section-heading">Shared Stories</h2>
+    <div class="col-lg-12 shared-experience">
+    <h2 class="section-heading">Shared stories</h2>
   </div>
   <div id="experience-carousel" class="carousel slide" data-ride="false" data-interval="false">
     <ol class="carousel-indicators">
-      <li data-target="#experience-carousel" data-slide-to="0" class="active"></li>
-      <li data-target="#experience-carousel" data-slide-to="1"></li>
-      <li data-target="#experience-carousel" data-slide-to="2"></li>
+      {% for story in stories %}
+      <li data-target="#experience-carousel" data-slide-to="{{ forloop.counter0 }}" class="{{ forloop.first|yesno:"active,," }}"></li>
+      {% endfor %}
     </ol>
     <div class="carousel-inner">
-      <div class="carousel-item active">
+      {% for story in stories %}
+      <div class="carousel-item {{ forloop.first|yesno:"active,," }}">
+        <a href="{% url 'main:single_story' story.uuid %}">
         <div class="row">
-        <img class="experience-image float-left d-block w-50" src="{% static "/images/animation_d.jpg" %}" width="150"
-          height="325"/ alt="First slide">
-        <div class="col-md-5">
-        <h3 class="experience-heading">Eating in a Restaurant</h3>
-        <p class="experience-content">...Waiting for the food felt like some kind of marathon and I just got so overwhelmed. But now that I’ve
-          realised hunger exacerbates my sensory differences I know how to cope. I try to make sure I’ve eaten something before I go out and a while back I bought some earplugs which are great for small and bustling places.. </p>
-        <em small class="experience-content">Sdy</em>
-         </div>
+          {% if story.image %}
+          <div class="experience-half w-50">
+          <div class="experience-image-box w-100">
+          <img class="experience-image d-block w-100" src="{% static "/images/" %}{{ story.image }}" alt="First slide" />
```

Comment:
If we think that the animations/images benefit users that require alt-text, then I think it would be good if we could also add the `alt` text into the `carousel.json` to have proper alt text instead of the place holder.

Having said that, I think we might just want to drop the alt-text, as I think the general recommendation is that if animations are just "visual fluff" and not necessary for the understanding then one should skip the alt-text as people using screenreader don't benefit from having these "flavour texts" read out.

Reply:
Ah, I'd planned to do something with these bug forgot. Thank you.

I've removed the alt tag text for now, but will happily put something in the json file for them if that's prefered?

Reply:
For now I'd say let's just go with the "no alt text" to not _spam_ screenreaders with it!